shared/profile/edit-avatar/index.desktop.tsx

```diff
@@ -298,8 +304,6 @@ class EditAvatar extends React.Component<_Props, State> {
             cursor: this.state.dragging ? '-webkit-grabbing' : 'default',
           },
         ])}
-        onMouseUp={this._onMouseUp}
-        onMouseDown={this._onMouseDown}
         onMouseMove={this._onMouseMove}
```

Comment:
Added this back so that when panning within the white modal box, the panning continues into the overlay area.